docs/usage/datamon_bundle_mount_new.md

```diff
@@ -20,6 +20,7 @@ datamon bundle mount new [flags]
       --destination string   The path to the download dir. Defaults to some random dir /tmp/datamon-mount-destination{xxxxx}
   -h, --help                 help for new
       --label string         The human-readable name of a label
+      --loglevel string      The logging level (default "info")
```

Comment:
Maybe the log level is best placed as a persistent flag under root?

Reply:
Yes agree: that is coming next. Deferred